.github/workflows/tests.yml

```diff
@@ -42,12 +42,11 @@ jobs:
 
     steps:
       - uses: actions/checkout@v2
-      - uses: actions/cache@v2
-        with:
-          path: ~/.isochrones
-          key: isochrones-${{ runner.os }}-${{ github.sha }}
-          restore-keys: |
-            isochrones-${{ runner.os }}-
+      - name: Download data files
+        run: |
+          wget -O ~/isochrones.zip "https://zenodo.org/record/4056544/files/isochrones.zip?download=1"
```

Comment:
I like the idea of keeping some test data on Zenodo, but this means we will have to remember to maintain this test data repo to make sure it's consistent with local data that gets auto-initialized.  Did you find that downloading this test dataset is necessary for the GH actions tests to run without timing out or something?

Reply:
Y'know I'm not entirely sure why I did this. I think it was just to speed things up and reduce bandwidth, but I'm not convinced that it was totally necessary!